api/resource.rb

```diff
@@ -198,10 +198,17 @@ def parameters
       (@parameters || []).reject(&:exclude)
     end
 
-    def exclude_if_not_in_version(version)
+    def below_version?(version)
```

Comment:
```suggestion
    def not_in_version?(version)
```

Nit: How about `not_in_version?`? Since resources are in or not in a version of a product, the method name will map a little nicer to common parlance. A method call will read as:

```ruby
object.not_in_version?(version)
```

Readers then won't need to make the connection themselves between `is_below_version?` and `exclude_if_not_in_version!` since they'll share a suffix.

Reply:
I'm not sure if `not_in_version` is misleading or not. Our version system has a distinct hierarchy such that a GA object would return `true` for `object.not_in_version('beta')`. I agree with consistency though. @rambleraptor  any thoughts on which is more clear?

Reply:
I'm realizing that 'below' and 'above' aren't really helpful concepts because it's not clear what position beta is in regards to ga.